api/api_logging.go

```diff
@@ -116,11 +117,28 @@ func encryptPassword(buf []byte) []byte {
 	var l string
 
 	for sc.Scan() {
+		match := ""
+		separator := ""
 		l = sc.Text()
-		match := `"password":"`
-		if strings.Contains(l, match) {
-			startIndex, endIndex, matchStrLen := FetchValueIndexForKey(l, match, `"`)
-			l = l[:startIndex+matchStrLen] + "****" + l[startIndex+matchStrLen+endIndex:]
+
+		filterString := `password|Cookie|Csrf-Token`
+		if present, _ := regexp.MatchString(filterString, l); present {
```

Comment:
The regex seems redundant here. You could remove the regex use a Boolean variable and set it to true if any of the switch conditions pass and use call code between line 138 to 141.
Incase u choose to keep the regex, try to check for err rather than ignoring it as _ .

Reply:
Directly using the switch will give more complexity, since only a very few lines in logs will pass this if condition, I will add a check for error in the regex instead of ignoring it.